pages/index.vue

```diff
@@ -3,3 +3,12 @@
     <h1 style="height: 5000px;">Home page</h1>
   </div>
 </template>
+
+<script setup lang="ts">
+definePageMeta({
+  hideBreadcrumbs: true
```

Comment:
..again, it feels like the breadcrumb component itself might make decision about hiding breadbrumb if only one level of route etc, rather than page defining it in the meta.